scripts/API/java.sh

```diff
@@ -1,16 +1,20 @@
 #!/bin/bash
 
+JAVA_DIR=${JAVA_DIR:-"iroha-java"}
+JAVA_REPO=${JAVA_REPO:-"https://github.com/hyperledger/iroha-java.git"}
+JAVA_BRANCH=${JAVA_BRANCH:-"iroha2-dev"}
```

Comment:
It's not used anywhere

Reply:
Good catch. I can barely think at this point.